cjs/html/style-element.js

```diff
@@ -1,16 +1,51 @@
 'use strict';
+const {parse} = require('cssom');
+
 const {registerHTMLClass} = require('../shared/register-html-class.js');
 
 const {TextElement} = require('./text-element.js');
 
 const tagName = 'style';
 
+const SHEET = Symbol('sheet');
+
 /**
  * @implements globalThis.HTMLStyleElement
  */
 class HTMLStyleElement extends TextElement {
   constructor(ownerDocument, localName = tagName) {
     super(ownerDocument, localName);
+    this[SHEET] = null;
+  }
+
+  get sheet() {
+    const sheet = this[SHEET];
+    if (sheet != null) {
```

Comment:
strict equality please, there's no way this can be *not* `null` at any time: it's either exactly `null` or whatever `parse(...)` returns, which I *hope* is never `null` or `undefined`! `!== null` might be better, thanks.

Reply:
P.S. apologies I've commented on the `cjs` file, I meant it on the `esm` one.